aclogview/CM_Examine.cs

```diff
@@ -57,9 +58,233 @@ public enum Enchantment_BFIndex {
+            node.Nodes.Add("_base_armor_bicep = " + _base_armor_bicep);
+            node.Nodes.Add("_base_armor_wrist = " + _base_armor_wrist);
+            node.Nodes.Add("_base_armor_hand = " + _base_armor_hand);
+            node.Nodes.Add("_base_armor_thigh = " + _base_armor_thigh);
+            node.Nodes.Add("_base_armor_shin = " + _base_armor_shin);
+            node.Nodes.Add("_base_armor_foot = " + _base_armor_foot);
+        }
     }
 
+    public class ArmorProfile
+    {
+        public float _mod_vs_slash;
+        public float _mod_vs_pierce;
+        public float _mod_vs_bludgeon;
+        public float _mod_vs_cold;
+        public float _mod_vs_fire;
+        public float _mod_vs_acid;
+        public float _mod_vs_electric;
+        public float _mod_vs_nether;
+
+        public static ArmorProfile read(BinaryReader binaryReader)
+        {
+            ArmorProfile newObj = new ArmorProfile();
+            newObj._mod_vs_slash = binaryReader.ReadSingle();
+            newObj._mod_vs_pierce = binaryReader.ReadSingle();
+            newObj._mod_vs_bludgeon = binaryReader.ReadSingle();
+            newObj._mod_vs_cold = binaryReader.ReadSingle();
+            newObj._mod_vs_fire = binaryReader.ReadSingle();
+            newObj._mod_vs_acid = binaryReader.ReadSingle();
+            newObj._mod_vs_electric = binaryReader.ReadSingle();
+            newObj._mod_vs_nether = binaryReader.ReadSingle();
+            return newObj;
+        }
+
+        public void contributeToTreeNode(TreeNode node)
+        {
+            node.Nodes.Add("_mod_vs_slash = " + _mod_vs_slash);
+            node.Nodes.Add("_mod_vs_pierce = " + _mod_vs_pierce);
+            node.Nodes.Add("_mod_vs_bludgeon = " + _mod_vs_bludgeon);
+            node.Nodes.Add("_mod_vs_cold = " + _mod_vs_cold);
+            node.Nodes.Add("_mod_vs_fire = " + _mod_vs_fire);
+            node.Nodes.Add("_mod_vs_acid = " + _mod_vs_acid);
+            node.Nodes.Add("_mod_vs_electric = " + _mod_vs_electric);
+            node.Nodes.Add("_mod_vs_nether = " + _mod_vs_nether);
+        }
+    }
+
+    public class HookAppraisalProfile
+    {
+        public uint _bitField;
+        public uint _validLocations;
+        public uint _ammoType;
+        public bool isInscribable;
+        public bool isHealer;
+        public bool isLockpick;
+
+        public static HookAppraisalProfile read(BinaryReader binaryReader)
+        {
+            HookAppraisalProfile newObj = new HookAppraisalProfile();
+            newObj._bitField = binaryReader.ReadUInt32();
+            newObj._validLocations = binaryReader.ReadUInt32();
+            newObj._ammoType = binaryReader.ReadUInt32();
+
+            if ((newObj._bitField & 0x1) != 0)
+                newObj.isInscribable = true;
+            if ((newObj._bitField & 0x2) != 0)
+                newObj.isHealer = true;
+            if ((newObj._bitField & 0x4) != 0)
+                newObj.isLockpick = true;

```

Comment:
These seem to match up with Enchantment_BFIndex - you should use i.e. Enchantment_BFIndex.BF_INSCRIBABLE instead of 0x1 etc.